core/ecschema-editing/src/Editing/Properties.ts

```diff
@@ -0,0 +1,271 @@
+    super(_schemaEditor);
+  }
+
+  /**
+   * Sets the extendTypeName attribute value.
+   * @param classKey The SchemaItemKey of the class.
+   * @param propertyName The name of the property.
+   * @param extendTypeName The extended type name of the property.
+   */
+  public async setExtendedTypeName(classKey: SchemaItemKey, propertyName: string, extendedTypeName: string) {
+    const property = await this.getProperty<MutablePrimitiveOrEnumPropertyBase>(classKey, propertyName);
+    property.setExtendedTypeName(extendedTypeName);
+  }
+
+  /**
+   * Sets the minLength attribute value.
+   * @param classKey The SchemaItemKey of the class.
+   * @param propertyName The name of the property.
+   * @param minLength The minimum length of the property.
+   */
+  public async setMinLength(classKey: SchemaItemKey, propertyName: string, minLength: number) {
+    const property = await this.getProperty<MutablePrimitiveOrEnumPropertyBase>(classKey, propertyName);
+    property.setMinLength(minLength);
+  }
+
+  /**
+   * Sets the maxLength attribute value.
+   * @param classKey The SchemaItemKey of the class.
+   * @param propertyName The name of the property.
+   * @param maxLength The maximum length of the property.
+   */
+  public async setMaxLength(classKey: SchemaItemKey, propertyName: string, maxLength: number) {
+    const property = await this.getProperty<MutablePrimitiveOrEnumPropertyBase>(classKey, propertyName);
+    property.setMaxLength(maxLength);
+  }
+
+  /**
+   * Sets the minValue attribute value.
+   * @param classKey The SchemaItemKey of the class.
+   * @param propertyName The name of the property.
+   * @param minValue The minimum value of the property.
+   */
+  public async setMinValue(classKey: SchemaItemKey, propertyName: string, minValue: number) {
+    const property = await this.getProperty<MutablePrimitiveOrEnumPropertyBase>(classKey, propertyName);
+    property.setMinValue(minValue);
+  }
+
+  /**
+   * Sets the maxValue attribute value.
+   * @param classKey The SchemaItemKey of the class.
+   * @param propertyName The name of the property.
+   * @param maxValue The maximum value of the property.
+   */
+  public async setMaxValue(classKey: SchemaItemKey, propertyName: string, maxValue: number) {
+    const property = await this.getProperty<MutablePrimitiveOrEnumPropertyBase>(classKey, propertyName);
+    property.setMaxValue(maxValue);
+  }
+}
+
+/**
+ * @alpha
+ * A class extending Properties allowing editing of PrimitiveProperty attributes.
+ */
+export class PrimitiveProperties extends PrimitiveOrEnumProperties {
+  public constructor(_schemaEditor: SchemaContextEditor) {
+    super(_schemaEditor);
+  }
+}
+
+/**
+ * @alpha
+ * A class extending Properties allowing editing of EnumerationProperty attributes.
+ */
+export class EnumerationProperties extends PrimitiveOrEnumProperties {
+  public constructor(_schemaEditor: SchemaContextEditor) {
+    super(_schemaEditor);
+  }
+}
```

Comment:
Not sure we really need to have own editors if we don't add any new values to it.
But if we keep them, we should then also have an explicit editor for Struct- and Navigation properties.

Reply:
I interpreted the code that we didn't add an editor for Navigation- and Struct-Properties because we don't want to allow to change what makes those two unique (struct/relationship reference), so the remain properties can be set through the common properties editor.

Reply:
There more just place holders for now.  We'll need them in the future, especially based on our conversation with Colin.  Nav and Struct will be added in the future too.